testsuite/src/test/resources/dwm/instance-0/jca.xml

```diff
@@ -0,0 +1,258 @@
+<deployment>
```

Comment:
well we can for instance-o because BeanValidation hasn't a jndi port set (using default 1099), but instance-1 has this setting using port 1199...I just believe it's clearer to have differnt xml files for each instance though. Maybe we can specify port 1099 here to make even more clear different settings, even if it's redundant